File: week15/week15-3.cpp
///week15-3.c
///LeetCode 1208,Get Equal Substrings within Budget寫到一半
int equalSubstring(char* s, char* t, int maxCost) {
    int ans = 0;

    int N = strlen(s); ///先知道字串的長度
    for(int i=0;i<N;i++){ ///字串的迴圈
        printf("%c %c\n",s[i],t[i] ); ///字串的陣列
    }
    return ans;
}

File: week15/week15-3.py
#week15-3.py 寫到一半的Python版,等一下還有"寫到一半的C版"
class Solution:
    def equalSubstring(self, s: str, t: str, maxCost: int) -> int:
        N =len(s)
        for i in range(N): #先用Python把字串的迴圈寫出來
            print(s[i],t[i])
        
        ans = 0
        return ans